# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ModuleMapKey.cpp
# Commit: c11cffaded9c
# Full Hash: c11cffaded9cfed53b04e5c7875278e9a9e55396
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 21:52:29
# Regressor Bug: 1742437
# File Overlap Count: 5
# Description:
#   Bug 1742437 - re-scope all generic module/script loader files under js/loader; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138133
# ==============================================================================

diff -r 0527d227c6b6 -r c11cffaded9c js/loader/ModuleMapKey.cpp
--- a/js/loader/ModuleMapKey.cpp	Fri Feb 18 12:45:52 2022 +0000
+++ b/js/loader/ModuleMapKey.cpp	Fri Feb 18 12:45:53 2022 +0000
@@ -4,7 +4,7 @@
 
 #include "ModuleMapKey.h"
 
-using namespace mozilla::dom;
+using namespace JS::loader;
 
 ModuleMapKey::ModuleMapKey(const nsIURI* aURI, nsIGlobalObject* aWebExtGlobal)
     : nsURIHashKey(aURI), mWebExtGlobal(aWebExtGlobal) {}
@@ -32,7 +32,7 @@
 PLDHashNumber ModuleMapKey::HashKey(KeyTypePointer aKey) {
   PLDHashNumber hash = nsURIHashKey::HashKey(aKey->mKey);
   // XXX This seems wrong.
-  hash =
-      AddToHash(hash, reinterpret_cast<uintptr_t>(aKey->mWebExtGlobal.get()));
+  hash = mozilla::AddToHash(
+      hash, reinterpret_cast<uintptr_t>(aKey->mWebExtGlobal.get()));
   return hash;
 }